staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -80,6 +80,14 @@ var (
 		},
 		[]string{"verb", "dry_run", "group", "version", "resource", "subresource", "scope", "component", "code"},
 	)
+	longRunningRequestsGauge = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Name:           "apiserver_longrunning_requests",
+			Help:           "Gauge of all active long-running apiserver requests broken out by verb, group, version, resource, scope and component. Not all requests are tracked this way. Use to replace apiserver_longrunning_gauge",
```

Comment:
I don't think we should change the help text, it would be best to mention that `apiserver_longrunning_requests` replaces the deprecated `apiserver_longrunning_gauge` metric in the changelog rather than in the help text.

Reply:
ok, will remove it.